.github/workflows/update_contributors.yml

```
@@ -0,0 +1,21 @@
+name: Update CONTRIBUTORS file
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```

Recommend to use `actions/checkout@v3` because of the node12 EOL on GitHub runners.

---

@tcraxs welcome back!:)
We are modern now btw and have a Matrix channel, the info how to join is in [README](https://github.com/ansible-collections/community.postgresql#communication) just in case you're interested:)